src/fsharp/FSharp.Build/Microsoft.FSharp.NetSdk.targets

```diff
@@ -37,8 +37,8 @@ WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and
 
   <PropertyGroup>
     <TargetProfile Condition=" '$(TargetFrameworkIdentifier)' == '.NETFramework' " >mscorlib</TargetProfile>
-    <TargetProfile Condition=" '$(TargetFrameworkIdentifier)' != '.NETFramework' AND ($(TargetFramework.StartsWith(netcoreapp1.)) OR $(TargetFramework.StartsWith(netstandard1.)))" >netcore</TargetProfile>
-    <TargetProfile Condition=" '$(TargetFrameworkIdentifier)' != '.NETFramework' AND !$(TargetFramework.StartsWith(netcoreapp1.)) AND !$(TargetFramework.StartsWith(netstandard1.))" >netstandard</TargetProfile>
+    <TargetProfile Condition=" '$(TargetFrameworkIdentifier)' != '.NETFramework' " >netcore</TargetProfile>
+    <TargetProfile Condition=" '$(TargetFrameworkIdentifier)' == '.NETStandard' AND '$(TargetFrameworkVersion)' >= 'v2.0'" >netstandard</TargetProfile>
```

Comment:
I don't think this will work - msbuild's >= only works if the string is an integer so you need to strip off the v from the version before doing the comparsion.